html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Html Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div class="coding-form">
+    <label for="email">Email</label><br />
+    <input type="text" name="email" id="email">
+  </div>
+  <div class="coding-form">
+    <label for="question">Why is coding cool?</label><br />
+    <select name="coding-question" id="question">
+      <option value="select-option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+  </div><br />
+  <div class="coding-form">
+    <label for="favorite-language">Favorite Language (choose one)</label><br /><br />
+    <input type="radio" name="favorite-language" value="html" checked="checked" />HTML
```

Comment:
each of these inputs should have a label, you can wrap the labels around the input if desired